docs/cli/edgedb_database/edgedb_database_wipe.rst

```diff
@@ -0,0 +1,32 @@
+.. _ref_cli_edgedb_database_wipe:
+
+
+====================
+edgedb database wipe
+====================
+
+Destroy the contents of a :ref:`database <ref_datamodel_databases>`
+
+.. cli:synopsis::
+
+    edgedb database wipe [<options>]
+
+
+Description
```

Comment:
Do you think it's worth noting that [it's equivalent to `reset schema to initial`](https://github.com/edgedb/edgedb-cli/blob/7292db83cf64b9eeb7ef10a83ef5262141fc466c/src/commands/database.rs#L64)?